src/GitHub.InlineReviews/Tags/InlineCommentTagger.cs

```diff
@@ -163,9 +167,16 @@ public IEnumerable<ITagSpan<InlineCommentTag>> GetTags(NormalizedSnapshotSpanCol
 
             if (relativePath != null)
             {
-                var liveFile = await sessionManager.GetLiveFile(relativePath, view, buffer);
-                fileSubscription = liveFile.LinesChanged.Subscribe(LinesChanged);
-                file = liveFile;
+                file = await sessionManager.GetLiveFile(relativePath, view, buffer);
+
+                var options = view.Options;
+                visibleSubscription =
+                    Observable.FromEventPattern<EditorOptionChangedEventArgs>(options, nameof(options.OptionChanged))
```

Comment:
This creates a subscription for when the `InlineCommentMarginVisible` editor option changes. There might be multiple events fired where the visibility hasn't changed. I was want one initial event initializing the visibility. Is this the cleanest/simplest way of doing that?